docs/data-masking-component-functions.md

```diff
@@ -0,0 +1,385 @@
+    | gen_rnd_email()                       |
+    +---------------------------------------+
+    | sma.jrts@example.com                  |
+    +---------------------------------------+
+    ```
+
+An example of `mask_pan(gen_rnd_pan())`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_pan(gen_rnd_pan());
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------+
+    | mask_pan(gen_rnd_pan())             |
+    +-------------------------------------+
+    | XXXXXXXXXXXX4444                    |
+    +-------------------------------------+
+    ```
+
+An example of `gen_rnd_us_phone()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_us_phone();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------+
+    | gen_rnd_us_phone()            |
+    +-------------------------------+
+    | 1-555-635-5709                |
+    +-------------------------------+
+    ```
+
+An example of `gen_rnd_ssn()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_ssn()
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-----------------------------+
+    | gen_rnd_ssn()               |
+    +-----------------------------+
+    | 995-33-5656                 |
+    +-----------------------------+
+    ```
+
+
+
+Use a selected dictionary to generate random terms. The dictionary must be loaded from a file with the following characteristics:
+
+
+* Plain text
+
+
+* One term per line
+
+
+* Must contain at least one entry
+
+Copy the dictionary files to a directory accessible to MySQL. The [secure-file-priv](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_secure_file_priv) option defines the directories where gen_dictionary_load() loads the dictionary files.
+
+!!! note
+
+    *Percona Server for MySQL* 8.0.21-12 enabled using the `secure-file-priv` option for gen_dictionary_load().
+
+    [MySQL 8.0.23](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-23.html) deprecated the `gen_blacklist()` function. Use `gen_blocklist()` instead.
+
+<table style="width:100%">
+    <tr>
+        <td>Parameter</td>
+        <td>Description</td>
+        <td style="width:30%">Returns</td>
+      
+    </tr>
+    <tr>
+        <td>gen_blocklist(str, dictionary_name, replacement_dictionary_name)</td>
+        <td>Replaces a term with a term from a second dictionary.</td>
+        <td>A dictionary term</td>
+           </tr>
+    <tr>
+        <td>gen_dictionary(dictionary_name)</td>
+        <td>Randomizes the dictionary terms</td>
+        <td>A random term from the selected dictionary.</td>
+        
+    </tr>
+    <tr>
+        <td>gen_dictionary_drop(dictionary_name)</td>
+        <td>Removes the selected dictionary from the dictionary registry.</td>
+        <td>Either success or failure</td>
+            </tr>
+    <tr>
+        <td>gen_dictionary_load(dictionary path, dictionary name)</td>
```

Comment:
No, the component uses tables, not files, and has different management functions.